vivisect/analysis/elf/libc_start_main.py

```diff
@@ -87,7 +87,7 @@ def prehook(self, emu, op, starteip):
         if op.iflags & envi.IF_CALL:
             # it's a call, get the target
             branches = [br for br in op.getBranches(emu) if not (br[1] & envi.BR_FALL)]
-            logger.debug('libc_start_main: 0x%x\ttgts: %r', self.startmain, branches)
+            logger.debug('libc_start_main: %r\ttgts: %r', [hex(x) for x in self.startmain], branches)
```

Comment:
this was a bug that plagued libc_start_main, and found it here.  close enough, so i just added it here.